doc/source/index.rst

```diff
@@ -2,161 +2,199 @@
-
-Contribution
-------------
-
-**Contributions** and **issue reports** are very welcome at `the github repository <https://github.com/arviz-devs/arviz>`_. We have a `contributing guide <https://github.com/arviz-devs/arviz/blob/main/CONTRIBUTING.md>`_ to help you through the process. If you have any doubts, please do not hesitate to contact us on `gitter <https://gitter.im/arviz-devs/community>`_.
-
-ArviZ's functions work with NumPy arrays, dictionaries of arrays, xarray datasets, and has built-in support for `PyMC3 <https://docs.pymc.io/>`_,
-`PyStan <https://pystan.readthedocs.io/en/latest/>`_, `CmdStanPy <https://github.com/stan-dev/cmdstanpy>`_,
-`Pyro <http://pyro.ai/>`_, `NumPyro <http://num.pyro.ai/>`_,
-`emcee <https://emcee.readthedocs.io/en/stable/>`_, and
-`TensorFlow Probability <https://www.tensorflow.org/probability>`_ objects. Support for Edward2 is on the roadmap.
-
-A Julia wrapper, `ArviZ.jl <https://julia.arviz.org/>`_ is
-also available. It provides built-in support for
-`Turing.jl <https://turing.ml/dev/>`_, `CmdStan.jl
-<https://github.com/StanJulia/CmdStan.jl>`_, `StanSample.jl
-<https://github.com/StanJulia/StanSample.jl>`_ and `Stan.jl <https://github.com/StanJulia/Stan.jl>`_.
-
-
-ArviZ is a non-profit project under NumFOCUS umbrella. If you want to **support ArviZ financially**, you can donate `here <https://numfocus.org/donate-to-arviz>`_.
-
-Citation
---------
-
-If you use ArviZ and want to **cite** it please use |JOSS|. Here is the citation in BibTeX format
-
-.. code:: bash
-
-    @article{arviz_2019,
-        doi = {10.21105/joss.01143},
-        url = {https://doi.org/10.21105/joss.01143},
-        year = {2019},
-        publisher = {The Open Journal},
-        volume = {4},
-        number = {33},
-        pages = {1143},
-        author = {Ravin Kumar and Colin Carroll and Ari Hartikainen and Osvaldo Martin},
-        title = {ArviZ a unified library for exploratory analysis of Bayesian models in Python},
-        journal = {Journal of Open Source Software}
-    }
+A Julia wrapper, `ArviZ.jl <https://julia.arviz.org/>`_ is also available. It provides built-in support for `Turing.jl <https://turing.ml/dev/>`_, `CmdStan.jl <https://github.com/StanJulia/CmdStan.jl>`_, `StanSample.jl <https://github.com/StanJulia/StanSample.jl>`_ and `Stan.jl <https://github.com/StanJulia/Stan.jl>`_.
+
+ArviZ's functions work with NumPy arrays, dictionaries of arrays, xarray datasets, and has built-in support for `PyMC3 <https://docs.pymc.io/>`_, `PyStan <https://pystan.readthedocs.io/en/latest/>`_, `CmdStanPy <https://github.com/stan-dev/cmdstanpy>`_, `Pyro <http://pyro.ai/>`_, `NumPyro <http://num.pyro.ai/>`_, `emcee <https://emcee.readthedocs.io/en/stable/>`_, and `TensorFlow Probability <https://www.tensorflow.org/probability>`_ objects. Support for Edward2 is on the roadmap.
+
+.. grid:: 2 2 3 4
+   :gutter: 3
+
+   .. grid-item-card:: Key Feature 1
```

Comment:
I like this style of key feature, not sure how it would look with longer titles. I would remove all 

Some of the things we could use:

* Interoperable

      Integrates with all major probabilistic programming libraries: 
      PyMC, CmdStanPy, PyStan, Pyro, NumPyro, emcee...

* Large suite of visualizations

      Provides over 25 plotting functions for all parts of Bayesian workflow:
      visualizing distributions, diagnostics, model checking...
      See the gallery for examples.

* State of the art diagnostics

      Latest published diagnostics and statistics are implemented, tested and distributed
      with ArviZ.

* Flexible model comparison

      Includes functions for comparing models with information criteria,
      and cross validation (both approximate and brute force).

* Built for collaboration

      Designed for flexible cross-language serialization using netCDF or Zarr formats.
      ArviZ also has a [Julia version](link somewhere) that uses the same {ref}`data schema <schema>`

* Labeled data

      Builds on top of xarray to work with labeled dimensions and coordinates

* Open source

      Distributed under the [Apache license](https://github.com/arviz-devs/arviz/blob/main/LICENSE),
      ArviZ is developed and maintained [publicly on GitHub](https://github.com/arviz-devs/arviz) 
      by a vibrant, responsive, and diverse {ref}`community <community>`

---

The last point is taken verbatim from numpy. And those are very clearly a starting point only.